configs/raspberrypi4cam_defconfig

```diff
@@ -0,0 +1,47 @@
+BR2_arm=y
+BR2_arm1176jzf_s=y
+BR2_TOOLCHAIN_BUILDROOT_GLIBC=y
+BR2_PACKAGE_HOST_LINUX_HEADERS_CUSTOM_4_19=y
+BR2_TOOLCHAIN_BUILDROOT_CXX=y
+BR2_TARGET_GENERIC_HOSTNAME="showmepicam"
+BR2_TARGET_GENERIC_ISSUE="Your webcam at your service!"
+BR2_INIT_SYSTEMD=y
+BR2_TARGET_GENERIC_ROOT_PASSWD="root"
+BR2_TARGET_GENERIC_GETTY_PORT="ttyAMA0"
+BR2_ROOTFS_OVERLAY="$(BR2_EXTERNAL_PICAM_PATH)/board/raspberrypi0cam/rootfs"
+BR2_ROOTFS_POST_BUILD_SCRIPT="$(BR2_EXTERNAL_PICAM_PATH)/board/raspberrypi0cam/post-build.sh"
+BR2_ROOTFS_POST_IMAGE_SCRIPT="$(BR2_EXTERNAL_PICAM_PATH)/board/raspberrypi0cam/post-image.sh"
+BR2_ROOTFS_POST_SCRIPT_ARGS="--configure-picam"
+BR2_LINUX_KERNEL=y
+BR2_LINUX_KERNEL_CUSTOM_TARBALL=y
+BR2_LINUX_KERNEL_CUSTOM_TARBALL_LOCATION="$(call github,raspberrypi,linux,4f2a4cc501c428c940549f39d5562e60404ac4f7)/linux-4f2a4cc501c428c940549f39d5562e60404ac4f7.tar.gz"
+BR2_LINUX_KERNEL_USE_CUSTOM_CONFIG=y
+BR2_LINUX_KERNEL_CUSTOM_CONFIG_FILE="$(BR2_EXTERNAL_PICAM_PATH)/board/raspberrypi0cam/linux-$(BOARD_NAME).config"
```

Comment:
```suggestion
BR2_ROOTFS_POST_BUILD_SCRIPT="$(BR2_EXTERNAL_PICAM_PATH)/board/raspberrypi0cam/post-build.sh"
BR2_ROOTFS_POST_IMAGE_SCRIPT="$(BR2_EXTERNAL_PICAM_PATH)/board/raspberrypi0cam/post-image.sh"
BR2_LINUX_KERNEL_CUSTOM_CONFIG_FILE="$(BR2_EXTERNAL_PICAM_PATH)/board/raspberrypi0cam/linux-$(BOARD_NAME).config"
```
`post-image.sh` use basename to build an appropriate image. For example
raspberrypi0cam/post-image.sh -> build image for **raspberrypi0cam** which define in raspberrypi0cam_defconfig
raspberrypi0/post-image.sh        -> build image for **raspberrypi0**        which define in raspberrypi0_defconfig
I think it is more understandable.

I see raspberrypi4cam_defconfig is slightly different from raspberrypi0cam_defconfig, what do you think if adding these changes programmatically  in build.sh?

Reply:
The `raspberrypi4cam_defconfig` is derived from the `bcm2711_defconfig`, which is the default kernel configuration for the Pi4. I merged some of the changes from the pi0cam config; perhaps it would be better to have the changes relative to the board specific configs?

Reply:
@osresearch
That is what I did on this [commit](https://github.com/showmewebcam/showmewebcam/pull/12/commits/3bcbe30fc8d8a3badf3629f9c8272bb80b61587c#).
I copy raspberrypi0cam_defconfig, then change path and dts file.
How do you think about this approach?